aces/Loris
# Rename webdriver package in composer.json

## Brief summary of changes

The `facebook/webdriver` package has been changed/renamed to `php-webdriver/webdriver` and as a result our current composer set up is causing errors and warnings in some cases.

This PR corrects the package name. It also uses a tagged release from that repo instead of relying on the master branch, which seems to cause issues on composer's side (see https://github.com/composer/composer/issues/7940)


#### Testing instructions (if applicable)

1. Checkout this branch, try running `make dev`.

**If you get an error about PHP and curl**, make sure you have the CURL extension enabled for your command line version of PHP.
    - To fix this on my Ubuntu VM, I ran: `sudo apt install php7.3-curl` and modified `vim /etc/php/7.3/cli/php.ini` by searching for "curl" and uncommenting the line I found.

**If you get an error from composer about 'unpushed changes'**, try:
- make clean
- composer remove facebook/webdriver
- composer update php-webdriver/webdriver

#### Link(s) to related issue(s)

* Reported in #6185 (but this does not resolve it as there is another related error)
* Resolves #6220
